modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss

```diff
@@ -22,6 +22,10 @@
 		margin-top: 24px;
 	}
 
+	.ddm-form-web__show-partial-results-alert {
```

Comment:
Hi @miltonmc, you've may noticed that this class is repeated on _portlet-forms.scss file. I didn't know how to avoid this. I thought in adding the style inside the jsp but I don´t know if it's the best choice. Do Please, could you give me a suggestion?